contact_us/index.html

```diff
@@ -13,7 +13,7 @@ <h1 class="pt-5 font-weight-bold">Contact us</h1>
   </section>
   <!-- //Header -->
 
-  <section class="container pb-5 pt-5 mb-5" style="height: 100%;">
+  <section class="container pb-5 pt-5 " style="height: 100%;">
```

Comment:
Well done! @VishwaAmarasinghe. Thanks for contributing

```suggestion
  <section class="container pb-5 pt-5">
```

We don't need this `style` attribute for the section. Could you please remove it also?

Reply:
ok brother